### DISCUSSION TITLE
Account creation

### DISCUSSION THREAD

**Discussion by danceshorribly** [jamieblomerus/WP-Mobile-BankID-Integration#13] [General]
Hi!

What happens when a user logs in via BankID? Is an account created?
Or would it be possible to create an account using the bankid as a login/register method?

**Comment by jamieblomerus** [comment]
I am pretty sure that there is an option/setting to allow users to register themselves through Mobile BankID.

**Reply by danceshorribly** [reply]
Yes, for sure, there is! I was probably unclear. The account created, it won't really have any data to it more than the personal number?
But the user could if they wanted to add email, password, name etc?

**Reply by jamieblomerus** [reply]
The account created is a fully normal WordPress account. So yes, one could add an email address, password etc. if one wants to.

Name is already added automatically when a user creates their account through Mobile BankID.
